saleor/checkout/calculations.py

```diff
@@ -73,13 +74,16 @@ def checkout_total(
 
 
 def checkout_line_total(
-    *, line: "CheckoutLine", discounts: Optional[Iterable[DiscountInfo]] = None
+    *,
+    line: "CheckoutLine",
+    discounts: Optional[Iterable[DiscountInfo]] = None,
+    channel: "Channel"
```

Comment:
What do you think do pass here `channel_id` instead of channel object?

Reply:
In this case, we can't do it because we pass the channel to the plugin. Some plugin might to require channel slug and others can require channel id.